ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -700,10 +700,11 @@ def test_tensorflow_reduce_mean(
 # reduce_variance
 @handle_frontend_test(
     fn_tree="tensorflow.math.reduce_variance",
-    dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+    dtype_and_x=statistical_dtype_values(
```

Comment:
Just need some context about this change :)

Reply:
The `statistical_dtype_values` helper is written specifically for testing statistical functions. Internally, it also calls `dtype_and_values` but sets the values for the safety factors first based on which statistical function is being tested. The idea is to reduce the range of values generated to prevent overflow errors.